packages/contractkit/src/wrappers/Attestations.ts

```diff
@@ -456,6 +456,28 @@ export class AttestationsWrapper extends BaseWrapper<Attestations> {
     )
   }
 
+  /**
+   * Updates sender's approval status in regards to allowing an attestation identifier
+   * mapping to be transfered from one address to another.
+   * @param identifier The identifier for this attestation.
+   * @param index The index of the account in the accounts array.
+   * @param from The current attestation address to which the identifier is mapped.
+   * @param to The new address to map to identifier.
+   * @param status The approval status
+   */
+  async approveTransfer(
+    identifier: string,
+    index: number,
+    from: Address,
+    to: Address,
+    status: boolean
+  ) {
+    return toTransactionObject(
```

Comment:
we're not doing this in other places, but I'd imagine we'd like to throw an error here if we can predict an on-chain revert. I'd like to do this, but I need to know how to access the address of `msg.sender` from here. Is that possible?

Reply:
Hm. I don't know if you can access `msg.sender` in the wrappers.

Reply:
I think it's ok to not do these kind of runtime checks. At the latest, gas estimation will show that

Reply:
If you are just "exposing" the method on the wrapper, you can use the higher order function `proxySend` here as well